cobra/cmd/license_gpl_3.go

```diff
@@ -18,23 +18,22 @@ package cmd
 func initGpl3() {
 	Licenses["gpl3"] = License{
 		Name:            "GNU General Public License 3.0",
-		PossibleMatches: []string{"gpl3", "gpl", "gnu gpl3", "gnu gpl"},
+		PossibleMatches: []string{"gpl3", "gplv3", "gpl", "gnu gpl3", "gnu gpl"},
 		Header: `
 This file is part of {{ .appName }}.
 
 {{ .appName }} is free software: you can redistribute it and/or modify
-it under the terms of the GNU Lesser General Public License as published by
+it under the terms of the GNU General Public License as published by
```

Comment:
Part of my PR in https://github.com/spf13/cobra/pull/356 was to avoid having parametrized appname in the notice which is a bummer for license detection. https://github.com/spf13/cobra/pull/356/files#diff-71e2b7d85e97ba305901d44ea0223e76R441 
I see you did not carry this around: any reason?